src/dune_config/dune_config.mli

```diff
@@ -120,4 +120,5 @@ val for_scheduler :
      t
   -> Dune_stats.t option
   -> insignificant_changes:[ `React | `Ignore ]
+  -> signal_watcher:[ `Yes | `No ]
```

Comment:
I'm not a big fan of booleans in general, but in that case does it make sense to use a plain `bool`? Or do we expect this to have more cases?